EIPS/eip-4844.md

```diff
@@ -54,6 +54,7 @@ Compared to full data sharding, this EIP has a reduced cap on the number of thes
 | `MAX_ACCESS_LIST_SIZE` | `2**24` |
 | `MAX_ACCESS_LIST_STORAGE_KEYS` | `2**24` |
 | `MAX_TX_WRAP_KZG_COMMITMENTS` | `2**12` |
+| `MAX_TX_WRAP_KZG_PROOFS` | `2**12` |
```

Comment:
This extra constant is not necessary, just use `MAX_TX_WRAP_KZG_COMMITMENTS`.

Reply:
updated :+1: